packages/orbit-design-tokens/src/dictionary/definitions/component/form-element/form-element-font-size.json

```diff
@@ -4,7 +4,7 @@
       "small": {
         "font-size": {
           "type": "size",
-          "value": "{foundation.font-size.normal}"
+          "value": "{foundation.font-size.small}"
```

Comment:
what's the purpose of having two size tokens with the same value? Below is a normal-size token with a normal value, let's then use the normal only and remove the small completely. Or having two different sizes

Reply:
If we don't find value with the token, we can consider removing it, rather than just changing its value. But this token is used in more places, so let's make sure we don't break anything with any change.

Reply:
this token actually not being used anywhere except tailwind config

Reply:
Right, but the deprecated one that referred to this is being used in Textarea, so it would lead to an erroneous change if the value was just changed. Removing the token and changing de replacement reference like you did is fail safe

Reply:
Also, the removal of such token and class is not a breaking change. Component specific tokens are undocumented and for internal use only, so they can be mentioned on the changelog, but we do not support them outside of Orbit. Therefore, their change / removal should not be documented as a breaking change